extensions/Khronos/KHR_materials_cmnConstant/README.md

```diff
@@ -0,0 +1,111 @@
+# KHR\_materials\_cmnConstant
+
+## Contributors
+
+* Don McCurdy, Google, <mailto:donmccurdy@google.com>
+
+## Status
+
+Draft (not ratified yet)
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines an unlit ("constant") shading model for use in glTF 2.0 materials, as an alternative to the Physically Based Rendering (PBR) shading models provided by the core specification. In mobile devices with limited resources, unlit materials offer a performant alternative to higher-quality shading models.
+
+## Extending Materials
+
+The common Constant material is defined by adding the `KHR_materials_cmnConstant` extension to any glTF material. When present, the extension indicates that a material should use the Constant shading model. Unrelated default values in the default PBR shading model, such as `diffuseFactor`, should be ignored.
+
+```
+{
+    "materials": [
+        {
+            "name": "MyUnlitMaterial",
+            "emissiveFactor": [ 0.2, 0.1, 0.0 ],
+            "extensions": {
+                "KHR_materials_cmnConstant": {}
+            }
+        }
+    ]
+}
+```
+
+No properties are defined within the extension object — it is simply a boolean flag indicating use of an unlit shading model.
+
+### Terms
+
+
+
+### Definition
+
+The Constant material model describes a constantly shaded surface that is independent of lighting. The material is defined only by properties already
+present in the [glTF 2.0 material specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#material): no new properties are added by this extension.
+
+Additional properties are allowed, but may lead to undefined behaviour in conforming viewers.
+
+The material is intended for unlit shading. Color is calculated as:
+
+```
+color = <emissiveTerm> + <occlusionTerm> * <aL>
```

Comment:
Should an unlit mode include indirect light?  How indirect does the light need to be before it gets included?

Reply:
Can't think of any reason to diverge from the glTF 1.0 extension here, https://github.com/KhronosGroup/glTF/tree/master/extensions/Khronos/KHR_materials_common#constant